### ISSUE TITLE
CanJS.com audit

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#2751]
Go through the website and look for broken demos & links, missing images & videos, spelling & grammar issues, accessibility issues, poor performance, etc.

**Comment by 0joshuaolson1** [comment]
For testing accessibility, [plenty of tools](https://github.com/collections/web-accessibility) are a good starting point to fix that.
